compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -399,7 +399,9 @@ private void defineConstructs(BLangPackage pkgNode, SymbolEnv pkgEnv) {
                 pkgEnv.scope.define(unresolvedPkgAlias, symbol);
             }
         }
-        initPredeclaredModules(symTable.predeclaredModules, pkgNode.compUnits, pkgEnv);
+        if (!pkgEnv.scope.owner.pkgID.equals(PackageID.ANNOTATIONS)) {
```

Comment:
Isn't `pkgNode` here the node corresponding to `pkgEnv`? We can use `pkgNode.packageID` if so. Let's also flip the check to be on `PackageID.ANNOTATIONS`, to avoid NPEs in case the package ID is null (which ideally shouldn't be the case).

```suggestion
        if (!PackageID.ANNOTATIONS.equals(pkgNode.packageID)) {
```

Reply:
I will change

Reply:
Updated - 1bcbc241b6a4a3dd45a84775f63dd4337a778bcc